### ISSUE TITLE
Different OOS evaluation metrics, user specified

### ISSUE THREAD

**Issue by crahal** [RobustiPy/robustipy#13]
Pseudo R2 is the natural candidate as it applies to probabilities as opposed to categories for binary outcomes, and is nicely interpretable. Ideally a suite of things that work for both binary and continuous DVs are preferable, but we could have  a flag to check and catch warn then revert to a default if the specified option isn't amenable.

**Comment by dhvalden** [comment]
to me: This means Out of Sample Evaluation Metric

**Comment by crahal** [comment]
Charlie: Change bug in x-axis tick label bug for KL divergeance. Metrics to include:

1. Default to [Pseudo-R2](https://www.pnas.org/doi/full/10.1073/pnas.1915006117)
2. Include [RMSE](https://statisticsbyjim.com/regression/root-mean-square-error-rmse/)
3. Include [Cross Entropy Loss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html).

**Comment by crahal** [comment]
I've updated code in [b97259d](https://github.com/RobustiPy/robustipy/commit/b97259d741147ae97d175698cd2c0c974615468e) to fix the xaxis tick label of subfigure 'c.', but I'd like to leave this open if possible because I think we still need more than two\three metrics choices (I would like to get the IMV into this, for example).
